src/Components/Modules/Bots.cpp

```diff
@@ -187,19 +185,19 @@ namespace Components
 			int forward = Game::Scr_GetInt(0);
 			int right = Game::Scr_GetInt(1);
 
-			forward = std::clamp(forward, -128, 127);
-			right = std::clamp(right, -128, 127);
+			forward = std::clamp<int>(forward, std::numeric_limits<char>::min(), std::numeric_limits<char>::max());
+			right = std::clamp<int>(right, std::numeric_limits<char>::min(), std::numeric_limits<char>::max());
```

Comment:
This looks like typical C++ obfuscation to me, to make things more complicated than they actually are. I would maybe agree on way shorter defines like `CHAR_MIN`/`CHAR_MAX` (but even that feels like adding ascii soup without function)

Reply:
I follow this thought process:
-128 and 127 are non-portable, non-standard.
The project is c++ not c so I use numeric_limists. Also, it's easier to remember when typing code.
Also easier to remember in my opinion. All I have to remember is that in the standard namespace there is a class called numeric_limits and I can slap in any type I want and it will yield the min or max depending on the function.

Reply:
Is the comment supposed to be sarcastic or?